memor/response.py

```diff
@@ -89,148 +80,118 @@ def __init__(
                 _validate_date_time(date, "date")
                 self._date_created = date
 
-    def __eq__(self, other_response):
+    def __eq__(self, other_response: Any) -> bool:
         """
         Check responses equality.
 
         :param other_response: another response
-        :type other_response: Response
-        :return: result as bool
         """
         if isinstance(other_response, Response):
             return self._message == other_response._message and self._score == other_response._score and self._role == other_response._role and self._temperature == other_response._temperature and \
                 self._model == other_response._model and self._tokens == other_response._tokens and self._inference_time == other_response._inference_time
         return False
 
-    def __str__(self):
+    def __str__(self) -> str:
         """Return string representation of Response."""
         return self.render(render_format=RenderFormat.STRING)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         """Return string representation of Response."""
         return "Response(message={message})".format(message=self._message)
 
-    def __len__(self):
-        """
-        Return the length of the Response object.
-
-        :return: length of the Response object
-        """
+    def __len__(self) -> int:
+        """Return the length of the Response object."""
         if self._message is None:
             return 0
         return len(self.render(render_format=RenderFormat.STRING))
 
-    def __copy__(self):
-        """
-        Return a copy of the Response object.
-
-        :return: a copy of Response object
-        """
+    def __copy__(self) -> Any:
+        """Return a copy of the Response object."""
         _class = self.__class__
         result = _class.__new__(_class)
         result.__dict__.update(self.__dict__)
         return result
 
-    def copy(self):
-        """
-        Return a copy of the Response object.
-
-        :return: a copy of Response object
-        """
+    def copy(self) -> Any:
```

Comment:
Response

Reply:
Since it's in the same class it can not be used.